docs/plugins/markdown/revealjs/themes.md

```diff
@@ -0,0 +1,267 @@
+# Reveal.js Themes
+
+## `auto`
+
+> Based on theme mode.
+
+@slidestart
+
+## Slide Title
+
+A paragraph with some text and a [link](https://mister-hope.com)
+
+---
+
+## Highlight
+
+```js [2-4|1-5]
+const add = (a, b) => {
+  if (typeof b === 'undefined') return a + 1
+
+  return a + b
+}
+```
+
+@slideend
```

Comment:
This is a very good example, and perhaps the source code can be placed in a details container for users to view.
```md
::: detials source code
:::
```